xbmc/windowing/egl/EGLWrapper.cpp

```diff
@@ -22,6 +22,8 @@
 #ifdef HAS_EGL
 #include "utils/log.h"
 #include "EGLNativeTypeAndroid.h"
+#include "EGLNativeTypeAmlAndroid.h"
+#include "EGLNativeTypeRKAndroid.h"
```

Comment:
Well, no more, no less than EGLNativeTypeAndroid, really. Just following the convention of no ifdef here, apparently